### ISSUE TITLE
In CPTTextLayer when self.text is nil everything crashes

### ISSUE THREAD

**Issue by RomaLLL** [core-plot/core-plot#233]
Please see this SO topic,

http://stackoverflow.com/questions/32897088/update-os-x-to-el-capitan-and-core-plot-crashes

I also added (my temporal) solution to this problem 
please take a note  :)

Thank for you awesome project!
Have a great day!
-Roma

**Comment by eskroch** [comment]
This is a duplicate of issue #210. It is fixed on the release-2.0 branch.

**Comment by RomaLLL** [comment]
Issue #210 complains that text is not appears 
I complained abou a crash when text is nil
I am sorry and dont want to be annoying 
but please review this again and confirm.

**Comment by eskroch** [comment]
The crash log in the StackOverflow question points to adding a `nil` value to the array of graphics contexts. This is the problem addressed by issue #210. Can you test with the latest code from the release-2.0 branch?

**Comment by RomaLLL** [comment]
Unfortunately not today already 4 am.

But I reviewed then issue #210 and it was about NSAttributedFont or something... In my case the text is not exists at all because I am not using anything besides the line itself in the line graph. 

Thus I asked you to review and confirm, bit if you say it os the same issue I am okay with it :-) 

Keep the great work!

(Btw I tried to reimplement line graph using SpriteKit, but the performance was the same (cpu wise)).
I really like your implementation!
